src/libraries/System.Private.CoreLib/src/System/Runtime/Intrinsics/X86/AvxVnni.PlatformNotSupported.cs

```diff
@@ -0,0 +1,70 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Runtime.CompilerServices;
+
+namespace System.Runtime.Intrinsics.X86
+{
+    [CLSCompliant(false)]
+    public abstract class AvxVnni : Avx2
+    {
+        internal AvxVnni() { }
+
+        public static new bool IsSupported { [Intrinsic] get { return false; } }
+
+        public new abstract class X64 : Avx2.X64
+        {
+            internal X64() { }
+
+            public static new bool IsSupported { [Intrinsic] get { return false; } }
+        }
+
+        /// <summary>
+        /// __m128i _mm_dpbusd_epi32 (__m128i src, __m128i a, __m128i b)
+        /// VPDPBUSD xmm, xmm, xmm
```

Comment:
nit: Based on the JIT support, this (and the other APIs) should be:
```suggestion
        /// VPDPBUSD xmm, xmm, xmm/m128
```

Reply:
andusing `ymm/m256` for the `Vector256<T>` APIs